src/common/circle-series.component.ts

```diff
@@ -133,19 +133,45 @@ export class CircleSeriesComponent implements OnChanges {
           barVisible: false,
           gradientId,
           gradientFill,
-          gradientStops: this.getGradientStops(color)
+          gradientStops: this.getGradientStops(color),
+          min: d.min,
+          max: d.max
         };
       }
     }).filter((circle) => circle !== undefined);
   }
 
-  getTooltipText({ tooltipLabel, value, seriesName }): string {
+  getTooltipText({ tooltipLabel, value, seriesName }, min: any, max: any): string {
```

Comment:
min and max can be inside the brackets:

`getTooltipText({ tooltipLabel, value, seriesName, min, max }): string {`